update_conda_lock/push_updated_lock.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
If we make this file Python it can work on windows / mac / linux? We already require Python for other parts?

Reply:
The script itself should work on all these OSs but I'm not sure about `update_lock.py` on Windows with its `\r\n` newlines as there are some file manipulations involved.